manifests/server/options.pp

```diff
@@ -119,6 +127,19 @@
     fail("The zone_notify must be ${valid_zone_notify}")
   }
 
+  $valid_dnssec_validation = ['yes', 'no', 'auto', 'absent']
+  if $dnssec_validation != undef and !member($valid_dnssec_validation, $dnssec_validation) {
+    fail("The dnssec_validation must be ${valid_dnssec_validation}")
+  }
+
+  if $dnssec_validation != undef {
+    $real_dnssec_validation = $dnssec_validation
+  } elsif $::osfamily == 'RedHat' and $::operatingsystemmajrelease == 5 {
+    $real_dnssec_validation = 'absent'
```

Comment:
This is confusing to me. I would rather we just "fail", letting a user say one thing and then silently doing something else on a particular platform is surprising